gradle/validation/rat-sources.gradle

```diff
@@ -38,6 +38,9 @@ allprojects {
             // Don't check under the project's build folder.
             exclude project.buildDir.name
 
+            // Exclude eclipse settings.
+            exclude ".settings"
```

Comment:
Thanks for looking at this. Can you move this down to be with the other "exclude eclipse" section?